index.js

```diff
@@ -16,46 +20,71 @@ var defaultOptions = {
 module.exports = {
   name: 'ember-bootstrap',
 
-  included: function included(app) {
+  included(app) {
```

Comment:
I think we can, at least the stuff that is supported in node 4.x, which is the next version to 0.12. See http://node.green. I think Ember-cli and other addons are doing that transition now.